atomic_reactor/plugins/post_compare_components.py

```diff
@@ -0,0 +1,106 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from atomic_reactor.plugin import PostBuildPlugin
+from atomic_reactor.constants import (PLUGIN_COMPARE_COMPONENTS_KEY,
+                                      PLUGIN_FETCH_WORKER_METADATA_KEY)
+
+
+supported_types = ["rpm"]
```

Comment:
Usually a module-level constant variable would be named in UPPER_CASE_WITH_UNDERSCORES.
https://www.python.org/dev/peps/pep-0008/#constants

Also, here `["rpm"]` is mutable, whereas `("rpm",)` is immutable. Using the immutable style (i.e. a 1-tuple) might make it clearer that this is never updated at runtime.